.gitignore

```diff
@@ -3,3 +3,5 @@
 tmp/
 lib/.precomp/
 .precomp/
+# Emacs backups
+*~
```

Comment:
`.gitignore` is for project-specific files. If your editor leaves droppings all over, you can do something like `echo '*~'>>~/.gitignore && git config --global core.excludesfile ~/.gitignore` rather than address it in every project you contribute to. Hail vi.